doc/spec/package-lock.md

```diff
@@ -0,0 +1,273 @@
+# package-lock and npm-shrinkwrap
+
+`npm` can have one of two different lock files:
+
+* `package-lock.json`, which is ordinarily always present and is never published.
+* `npm-shrinkwrap.json`, which is created with `npm shrinkwrap` and usually published.
+
+You can only have one of them and in the event that you have both,
```

Comment:
I think it read clearer with the same meaning as "You should only have one of them" rather than "You can only have one of them"